test/runtests.sh

```diff
@@ -1,13 +1,16 @@
 #!/usr/bin/env bash
 
+# change to script directory
+cd "${0%/*}"
```

Comment:
Yes. This switches the current working directory to the directory in which the script is in, as per http://stackoverflow.com/a/16349776/313758